corehq/motech/repeaters/templates/repeaters/partials/attempt_history.html

```diff
@@ -18,13 +18,13 @@
                         <div>
                             <strong>Attempt #{{ attempt_number }}</strong> -
                             {% if attempt.state == 'SUCCESS' %}
-                                <i class="fa fa-check"></i>
+                                <br/><i class="fa fa-check"></i>
                             {% elif attempt.state == 'EMPTY' %}
-                                <i class="fa fa-check"></i>
+                                <br/><i class="fa fa-check"></i>
                             {% elif attempt.state == 'CANCELLED' %}
-                                <i class="fa fa-remove"></i> {% trans "Cancelled" %}
+                                <br/><i class="fa fa-remove"></i> {% trans "Cancelled" %}
```

Comment:
You can drop lines 24 and 25.

Line 26 correctly catches the "CANCELLED" state, and correctly excludes the word "Cancelled".

Reply:
(Although it's not in [the original branch that got merged in](https://github.com/dimagi/commcare-hq/pull/32993/files), lines 20 and 22 can be `or`ed, because lines 21 and 23 are identical.)

Reply:
Cool. Updated

Reply:
Awesome! Thank you. I think that's everything.